### PR TITLE
Improve dragging experience

### PR DESCRIPTION
**Pull request by saghul**
The entire page is draggable, except:

- buttons
- any input type
- the filmstrip

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
         body {
             -webkit-app-region: drag
         }
+        button,
+        input[type] {
```
Comment (AkshitKrNagpal) [inline app/index.html]: Why `input[type]` ?
Why not just `input`?

Comment (saghul) [reply app/index.html]: It matches only inputs which have a type attribute. TBH I'm not sure that's enough, but it seems to work well in practice 😄

File: app/features/conference/components/Conference.js [hunk 2]
```diff
@@ -208,6 +207,17 @@ class Conference extends Component<Props, *> {
      * @returns {void}
      */
     _onVideoConferenceJoined(conferenceInfo: Object) {
+        // Inject some style into the iframe so everything except the filmstrip,
+        // buttons or any input is draggable.
+        const iframe = this._api.getIFrame();
+        const injectStyle = document.createElement('style');
+
+        injectStyle.type = 'text/css';
```
Comment (hristoterezov) [inline app/features/conference/components/Conference.js]: Isn't it going to be better if you put this code into separate function?

I think this is good enough for temporary solution but isn't it going to be better if we implement more generic way to inject styles into meet from the iframe api?

File: app/index.html (hunk 1 above)
Comment (hristoterezov) [reply app/index.html]: But something is not working if we use just input?

Comment (saghul) [reply app/index.html]: I don’t know, we can change it.

File: app/features/conference/components/Conference.js (hunk 2 above)
Comment (saghul) [reply app/features/conference/components/Conference.js]: I can move to a function if that feels better.

As for having an API, I don’t disagree, but that isn’t orthogonal to this. We can use that, once / if we have it.

**Comment by saghul** [comment]
What I’m trying to solve is providing a good dragging expresidente in the entire window. She the entire window is draggable, you cannot select text in a text field, for example. Also, you can’t scroll the chat or the filmstrip by clicking on the scrollbar. That is what this PR fixes.

File: app/features/conference/components/Conference.js (hunk 2 above)
Comment (hristoterezov) [reply app/features/conference/components/Conference.js]: sure!

**Comment by saghul** [comment]
@hristoterezov Updated, PTAL!

**Pull request merged**